dipy/direction/tests/test_bootstrap_direction_getter.py

```diff
@@ -21,7 +21,7 @@
 
 
 def test_bdg_initial_direction():
-    """This test the number of inital direction."
+    """This tests the number of initial directions."
```

Comment:
not sure about this one

Reply:
meaning `test` singular

Reply:
I can confirm that the proposed change is 👍

Reply:
:+1: Strictly speaking, I'd say that test methods do not require docstrings. Additionally, if I'm not mistaken PEP8 discourages starting docstrings with wording like "This method/class" etc. At the same time, if the naming or the test contents are not clear enough (which are indicative of bad naming wrt the acronyms/context of the package, or too many things done in a test), I'd opt for a simple comment like `# Test the number of initial directions`. But that is beyond the scope of this PR.

Thanks both for the reviews.